gen/internal/tests/thrift/nozap.thrift

```diff
@@ -0,0 +1,17 @@
+enum EnumDefault {
+    Foo, Bar, Baz
+}
+
+struct PrimitiveRequiredStruct {
+    1: required bool boolField
+    2: required byte byteField
+    3: required i16 int16Field
+    4: required i32 int32Field
+    5: required i64 int64Field
```

Comment:
I would make some of these fields containers so that it would *normally* generate zap wrappers for them.